Realm/RLMObjectSchema.mm

```diff
@@ -171,6 +171,24 @@ + (instancetype)schemaForObjectClass:(Class)objectClass {
     return schema;
 }
 
++ (bool)_isDisallowedLazyProperty:(NSString *)propertyName
+                     ignoredProps:(NSArray *)ignoredProperties
+                      snippedName:(NSString **)snippedName {
+    // A Swift lazy var shows up as two separate children on the reflection tree: one named 'x', and another that is
```

Comment:
It might be clearer for this method to become `+baseNameForLazySwiftProperty:` that returns the base property name or nil when it's not the name of the storage of the lazy Swift property, and to have `+propertiesForClass:isSwift:` handle checking if it's ignored and throwing the exception.

If that's not an improvement, I'd suggest `s/ignoredProps/ignoredProperties/` and `s/snippedName/basePropertyName/`.